src/CollisionHandler.cpp

```diff
@@ -0,0 +1,57 @@
+#include "CollisionHandler.h"
+
+CollisionHandler::CollisionHandler() {
+	this->quadtreeMov = new Quadtree(0, {0,0,2000,2000});
+	this->quadtreePro = new Quadtree(0, {0,0,2000,2000});
+	this->quadtreeDam = new Quadtree(0, {0,0,2000,2000});
+}
+
+CollisionHandler::~CollisionHandler() {
```

Comment:
props to @JohnAgapeyev  for catching this

there's a memory leak as the destructor never frees the quadtreeMov  quadtreePro  quadtreeDam